mods/common/chrome/editor.yaml

```diff
@@ -584,23 +586,6 @@ Container@EDITOR_WORLD_ROOT:
 			TooltipContainer: TOOLTIP_CONTAINER
 			Font: Bold
 			Key: f1
-		Label@ZOOM_LABEL:
-			X: 495
-			Width: 50
-			Height: 25
-			Text: Zoom:
-			Align: Right
-			Font: Bold
-			Contrast: true
-		DropDownButton@ZOOM_BUTTON:
```

Comment:
Specific named zoom levels don't make a lot of sense now, so this is instead moved to the mouse wheel.

Reply:
This creates a gap in the button row, so the undo/redo buttons should be moved to the left.